hail/python/hailtop/utils/utils.py

```diff
@@ -227,3 +229,31 @@ def is_transient_error(e):
             if msg:
                 log.info(msg(exc), exc_info=True)
         await sleep_and_backoff(delay)
+
+
+async def retry_long_running(name, f, *args, **kwargs):
+    delay_secs = 0.1
+    while True:
+        try:
+            start_time = time_msecs()
+            await f(*args, **kwargs)
```

Comment:
do you need a return here?

Reply:
Hmm.  Maybe.  This is to retry long running things, and by long running, I mean things that we expect will never return.  But I guess they return, we should to.  Yes, I'll add it.  Good suggestion.